modules/candidate_list/templates/menu_candidate_list.tpl

```diff
@@ -219,7 +219,9 @@
 <script>
 var table = RDynamicDataTable({
     "DataURL" : "{$baseurl}/candidate_list/?format=json",
+    {if $access_all_profiles}
```

Comment:
even if user doesn't have this permission, we still need to have this script because it deals with the column formatting for the feedback column. Look at the file ```modules/candidate_list/js/columnFormatter.js``` this is where the formatting happens